MediaGalleryUi/view/adminhtml/web/js/directory/directoryTree.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.g
+ * See COPYING.txt for license details.
+ */
+
+/* global FORM_KEY */
+define([
+    'jquery',
+    'uiComponent',
+    'jquery/jstree/jquery.jstree'
+], function ($, Component) {
+    'use strict';
+
+    return Component.extend({
+        defaults: {
+            filterChipsProvider: 'componentType = filters, ns = ${ $.ns }',
+            directoryTreeSelector: '#media-gallery-directory-tree',
+            /* eslint-disable max-len */
+            urlProvider: 'name = media_gallery_listing.media_gallery_listing.media_gallery_columns.thumbnail_url, ns = media_gallery_listing',
+            options: {
+                treeInitData: []
```

Comment:
Why do we need `options` parent node? Is there any other options that we might use?